tutorials-v5/time-evolution/010_brmesolve_phonon_interaction.md

```diff
@@ -213,8 +213,8 @@ def brme_step(args):
     ).expect[0][-1]
 
 
-# use QuTiP's builtin parallelized for loop, parfor
-results = parfor(brme_step, itertools.product(wd_list, Om_list))
+# use QuTiP's builtin parallelized for loop: parallel_map
+results = parallel_map(brme_step, list(itertools.product(wd_list, Om_list)))
```

Comment:
It would be nice if this example should be using the class interface instead of the function call in v5.
The class interface is to be used when computing the same system with different parameters, which is what is done here.

Reply:
Thanks for the review @Ericgig! I changed from using `brmesolve` to `BRSolver` in the last commit. `BRSolver` currently requires that the spectra for the `a_ops` are given as `coefficient`. Hence, I can not pass a string spectrum.

I think it would be great if `BRSolver` also accepts other types of spectra.